apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -1122,3 +1130,24 @@ func (r *OpenStackControlPlane) ValidateTopology(basePath *field.Path) *field.Er
 	}
 	return nil
 }
+
+func (r *OpenStackControlPlane) ValidateNotificationsBusInstance(basePath *field.Path) *field.Error {
```

Comment:
@gibizer this should help to clarify the question about empty string or malformed input mentioned in another comment.
I can add envTests to validate this function (I'll do in my next iteration)